packages/manager/src/features/OneClickApps/oneClickApps.ts

```diff
@@ -2139,6 +2249,28 @@ export const oneClickApps: OCA[] = [
       'All-in-one database deployment, management, and monitoring system.',
     website: 'https://docs.splunk.com/Documentation/Splunk',
   },
+  {
+    alt_description:
+      'A private by design messaging platform.',
+    alt_name: 'Anonymous messaging platform.',
+    categories: ['Productivity'],
+    colors: {
+      end: '70f0f9',
+      start: '11182f',
+    },
+    description: `SimpleX Chat - The first messaging platform that has no user identifiers of any kind - 100% private by design. SMP server is the relay server used to pass messages in SimpleX network. XFTP is a new file transfer protocol focussed on meta-data protection. This One-Click APP will deploy both smp and xftp servers.`,
```

Comment:
Not sure if these need to be capitalized, please ignore if so!
```suggestion
    description: `SimpleX Chat - The first messaging platform that has no user identifiers of any kind - 100% private by design. SMP server is the relay server used to pass messages in SimpleX network. XFTP is a new file transfer protocol focussed on meta-data protection. This One-Click APP will deploy both SMP and XFTP servers.`,
```